app/src/components/widgets/charts/cumulativeTrendChart/cumulativeTrendChartJs.jsx

```diff
@@ -0,0 +1,164 @@
+import { PureComponent } from 'react';
+import { injectIntl, intlShape } from 'react-intl';
+import PropTypes from 'prop-types';
+import { ChartJS } from 'components/widgets/charts/common/chartjs';
+import classNames from 'classnames/bind';
+import styles from './cumulativeTrendChart.scss';
+import { getChartData } from './chartjsConfig';
+import { CumulativeChartLegend } from './cumulativeChartLegend';
+
+const cx = classNames.bind(styles);
+
+@injectIntl
+export class CumulativeTrendChartJs extends PureComponent {
```

Comment:
You already have `CumulativeTrendChart`. What is the purpose to create a new one?